tfx/components/example_gen/utils.py

```diff
@@ -275,6 +291,20 @@ def _verify_split_pattern_specs(
   return is_match_span, is_match_date, is_match_version
 
 
+def _verify_range_config(
```

Comment:
Let's call out the restriction on `RangeConfig` in the component interface. I was thinking that we get the latest span that is inside the range specified by the `RangeConfig` until see the code here.

@1025KB  @hughmiao any context we go with the decision to allow only `start_span_number == end_span_number`? Another question will be, how do we support other range config types in ExampleGen?

Reply:
will move this check to the component interface. the 2nd portion was partially resolved at our sync, but i will leave open in case more discussion is needed.